src/symbols.c

```diff
@@ -2289,6 +2290,32 @@ static gint get_fold_header_after(ScintillaObject *sci, gint line)
 }
 
 
+/* returns the line after following all brace match for @brace on @line */
+static gint resolve_matching_braces(ScintillaObject *sci, gint line, gint brace)
+{
+	gint pos = sci_get_position_from_line(sci, line);
+	gint line_end = sci_get_line_end_position(sci, line);
+	gint lexer = sci_get_lexer(sci);
```

Comment:
I can, I just wanted to see what you guys thought before making a PR that will have to be changed a bunch of times, and put it here with the current discussion (which is available in the comments mailing list for posterity and emailed to anyone interested as well) rather than spreading it all over.